Save section/block preferences in user data (#2148)
woocommerce/wc-admin: client/dashboard/README.md

Dashboard
=======

This folder contains the components used in the Dashboard page.

## Extending the Dashboard

New Dashboard sections can be added by hooking into the filter `woocommerce_dashboard_default_sections`. For example:

```js
addFilter( 'woocommerce_dashboard_default_sections', sections => {
	return [
		...sections,
		{
			key: 'example',
			component: ExampleSection,
			title: 'My Example Dashboard Section',
			isVisible: true,
			icon: 'arrow-right-alt',
			hiddenBlocks: [],
		},
	];
} );
```

> Each report is defined by an object containing `key`, `title`, `component` and `hiddenBlocks`.

Reviewer: Should this say `Each section is ...`

> Each report is defined by an object containing `key`, `title`, `component` and `hiddenBlocks`.

Reviewer 2: ```suggestion
Each report is defined by an object containing the following properties.
```
